test/jdk/jdk/jfr/event/gc/detailed/TestGCLockerEvent.java

```diff
@@ -26,6 +26,7 @@
  * @test TestGCLockerEvent
  * @key jfr
  * @requires vm.hasJFR
+ * @requires vm.gc == "G1" | vm.gc == null
```

Comment:
I think this should instead be `@requires vm.gc.G1`.  The null check is wrong if running with a build that doesn't include G1 and the default GC is being used.

Reply:
True, it does look suspicious. I matched what other tests in the same folder are doing. Maybe we should fix them all at once in the followup PR?

Reply:
I don't see a reason to change this test from being wrong to being differently wrong.

Reply:
Yeah, you are right. Changed, re-tested. See new commit.